api/specs/webserver/scripts/openapi_node_meta.py

```diff
@@ -0,0 +1,45 @@
+""" Helper script to automatically generate OAS
+
+This OAS are the source of truth
+"""
+
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import json
+
+import jsonref
+from fastapi import FastAPI
+from models_library.services import ServiceDockerData
+
+app = FastAPI(redoc_url=None, openapi_version="3.0.0")
+
+
+@app.get(
+    "",
+)
+async def get_project_inputs(project: ServiceDockerData):
+    """New in version *0.10*"""
+
+
+if __name__ == "__main__":
+    from _common import CURRENT_DIR, create_openapi_specs
+
+    #  Generate OAS for the Project pydantic model via the FastAPI app
+    #  NOTE: currently not used, as the generated OAS does not generate x-pattern properties
+    #  it has problem with types ex. ServiceInputsDict = dict[ServicePortKey, ServiceInput], where key is a constrained string
+    create_openapi_specs(
+        app, CURRENT_DIR.parent / "../common/schemas/openapi-node-meta-generated.yaml"
+    )
```

Comment:
So this is just generated and stored without using it? Did I get ti right? If it has no further use I would not keep it.

Reply:
yes correct, I have already deleted it in my last commit. It would be nice someday the generated open api, but for now it is generated from the generated json schema